spec/capitalize_decorator_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './decorator'
+require './capitalize_decorator'
+
+describe CapitalizeDecorator do
+  let(:decorator) { CapitalizeDecorator.new(double('decorated object')) }
+
+  describe '#correct_name' do
+    it 'capitalizes the name returned by the decorated object' do
+      allow(decorator).to receive(:correct_name).and_return('Benawi')
+
+      expect(decorator.correct_name).to eq('Benawi')
+    end
```

Comment:
- [ ] Kindly make understand how you are testing an already capitalized word. I expected to see a name with a small letter to equal the capitalized same name.
- [ ] Kindly test the trimmer_decorator method as well.